api_testing/test_updating_resource.py

```diff
@@ -0,0 +1,18 @@
+from api_testing.credentials import CredentialUpdating
+
+
+def test_update_user_199(session, base_url):
+    update_user_10 = session.put(base_url + '199', json=CredentialUpdating.user_199)
+    assert update_user_10.status_code == 200
+    assert dict(update_user_10.json()).get('id') == 199
```

Comment:
You could add checks for **userId**, **id**, **title** and **completed** response fields:

```
def test_update_user_199(session, base_url):
    res = session.put(base_url + '199', json=CredentialUpdating.user_199)
    assert res.status_code == 200
    r_json = res.json()
    assert r_json['userId'] == CredentialUpdating.user_199['userId']
    assert r_json['id'] == CredentialUpdating.user_199['id']
    assert r_json['title'] == CredentialUpdating.user_199['title']
    assert r_json['completed'] == CredentialUpdating.user_199['completed']
```